src/app/modules/item/pages/item-log-view/item-log-view.component.ts

```diff
@@ -4,27 +4,13 @@ import { ActivityLog, ActivityLogService } from 'src/app/shared/http-services/ac
 import { Observable, ReplaySubject } from 'rxjs';
 import { distinct, switchMap, map } from 'rxjs/operators';
 import { mapToFetchState } from 'src/app/shared/operators/state';
+import { ItemType } from '../../../../shared/helpers/item-type';
 
-interface Column {
+export interface Column {
```

Comment:
At the end, this is not exported anymore. Is it?

Reply:
Fixed